astropy/vo/client/conesearch.py

```diff
@@ -81,7 +83,7 @@ def conesearch(center, radius, verb=1, **kwargs):
     Parameters
     ----------
     center : `~astropy.coordinates.SkyCoord`, `~astropy.coordinates.BaseCoordinateFrame`, or sequence of length 2
-        Position of the center of the cone to search.  It may be specified as an
+        Position of the center of the cone to search. It may be specified as an
```

Comment:
Normally double whitespace is used to highlight a new sentence. See also two lines later.